.circleci/config.yml

```diff
@@ -177,7 +177,7 @@ jobs:
             echo $GCLOUD_SERVICE_KEY > ~/gcloud-service-key.json
             gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
             export UNISWAP_CONTRACT_DATA="uniswap-contract-data.txt"
-            export BONDED_ECDSA_KEEP_VENDOR_CONTRACT_DATA="BondedECDSAKeepVendor.json"
+            export BONDED_ECDSA_KEEP_FACTORY_CONTRACT_DATA="BondedECDSAFactoryVendor.json"
```

Comment:
We need to have a reference to `BondedECDSAKeepFactory.sol`, not the vendor. Even though with the code here we can't update the vendor, we can still update the factory and the whole point is that Keep-org should not be able to do that for tBTC on mainnet.

Reply:
Nice catch! Seems I made an edit error :man_facepalming: